zookeeper/source/zk_nn_client.cc

```diff
@@ -664,6 +664,7 @@ namespace zkclient{
 
```

Comment:
As we discussed, it now prints out errors when something in multiop fails.

2e990fc